src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -6261,8 +6191,8 @@ void ReportThermalResilience(EnergyPlusData &state)
 
 void ReportCO2Resilience(EnergyPlusData &state)
 {
-    int NoBins = 3;
     if (state.dataHeatBalSurfMgr->reportCO2ResilienceFirstTime) {
+        int NoBins = 3;
```

Comment:
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:6264]:(style),[variableScope],The scope of the variable 'NoBins' can be reduced.
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:6355]:(style),[variableScope],The scope of the variable 'NoBins' can be reduced.